CodeStarterCamp_Week4/main.swift

```diff
@@ -8,12 +8,14 @@
 
 import Foundation
 
-var person = BodyCondition(startingPoint: 0)
+let hellExercise: [Exercise] = [sitUp, activeRest, deadLift, activeRest, plank, activeRest, barbellRow, activeRest, cablePullover, activeRest, squat, activeRest, lunge, activeRest, legExtension, activeRest, longRun]
 
-let exercise: [Exercise] = [sitUp, sitUp, sitUp, activeRest, deadLift, deadLift, deadLift, activeRest, plank, plank, plank, activeRest, barbellRow, barbellRow, barbellRow, activeRest, cablePullover, cablePullover, cablePullover, activeRest, longRun]
+let normalExercise: [Exercise] = [sitUp, activeRest, squat, activeRest, activeRest, barbellRow, activeRest, cablePullover, activeRest, legExtension, activeRest, longRun]
 
-let hellRoutine = Routine(routineName: "지옥의 헬스루틴", exercises: exercise)
+// [Routine]에 들어갈 루틴 선언
+let hellRoutine = Routine(routineName: "지옥의 헬스루틴", exercises: hellExercise)
+let normalRoutine = Routine(routineName: "보통의 헬스루탄", exercises: normalExercise)
 
-hellRoutine.startRoutine() // 지옥의 헬스루틴 시작
-person.checkYourCondition() // 인바디 측정 
+var yagomFitnessCenter = FitnessCenter(name: "야곰", listOfRoutine: [hellRoutine, normalRoutine])
+yagomFitnessCenter.exerciseRoutine()
```

Comment:
FitnessCenter 타입은 `클래스`로 구현했죠~?
클래스의 var 프로퍼티는, 타입의 인스턴스가 let 이어도 변경할 수 있어요. 😄 (많이 어려워 하셨던 부분!)

그니까 `yagomFitnessCenter`는 `상수(let)`로 저장해줘도, 전혀 문제 없이 작동합니다!

```suggestion
let yagomFitnessCenter = FitnessCenter(name: "야곰", listOfRoutine: [hellRoutine, normalRoutine])
yagomFitnessCenter.exerciseRoutine()
```